packages/client/src/index.ts

```diff
@@ -6,6 +6,7 @@ export * from './util/logger'
 export * from './util/escape'
 export * from './util/time'
 export * from './util/generics'
+export {collectAll} from './util/common'
```

Comment:
I think this can be avoid as a default export.
```suggestion
```

Reply:
this util function will be used in future PR for third party libraries but it can be also used on it's own to consume all query rows at once